### PR TITLE
prefetch screenshots on kifi.com

### PR DESCRIPTION
**Pull request by joonhocho**

### CODE REVIEW

File: kifi-backend/modules/shoebox/public/js/main.js [hunk 1]
```diff
@@ -2818,6 +2819,33 @@ $(function () {
 		});
 	}
 
+	function prefetchScreenshots(keeps) {
+		if (!(keeps && keeps.length)) {
+			return;
+		}
+		return $.postJson('https://www.kifi.com/site/keeps/screenshot', {
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/js/main.js:2826]: might want to use `xhrBase` here for consistency

**Comment by 2is10** [comment]
Cool! This will get the images into the browser cache, but a server roundtrip will still be required for each keep to fetch its screenshot image URL again, yes? We may eventually want to save those in each keep’s element data.

File: kifi-backend/modules/shoebox/public/js/main.js (hunk 1 above)
Comment (joonhocho) [inline kifi-backend/modules/shoebox/public/js/main.js:2826]: fixed! thanks.

**Comment by joonhocho** [comment]
Yup will address that issue someday. +1

**Pull request merged**